steps/gcp-step-disk-delete/spec.schema.json

```diff
@@ -0,0 +1,47 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+      "google": {
+          "type":"object",
+          "description":"A mapping of GCP account configuration.",
+          "properties": {
+              "service_account_info": {
+                  "type": "object",
+                  "x-relay-connectionType": "gcp",
+                  "description": "A Relay Google Cloud Platform (GCP) connection to use",
+                  "properties": {
+                    "service-account.json": {
```

Comment:
This is called `serviceAccountKey` in the real connection type, and it isn't Base64-encoded so the description needs to be updated (or omitted since the connection types are opaque).